src/commands/bridge/pause.ts

```diff
@@ -0,0 +1,56 @@
+import { KeyringPair } from '@polkadot/keyring/types'
+import { GluegunToolbox, filesystem } from 'gluegun'
+import {
+  EthereumConfig,
+  Network,
+  SubstrateConfig,
+} from '@buildwithsygma/sygma-sdk-core'
+import { Wallet } from 'ethers'
+import { InitializedWallets, RpcEndpoints } from '../../types'
+import {
+  sendEvmPauseTransaction,
+  sendSubstratePauseTransaction,
+} from '../../utils'
+
+module.exports = {
+  name: 'pause',
+  run: async (toolbox: GluegunToolbox) => {
+    const { sharedConfig, wallet } = toolbox
+
+    const rawConfig = await sharedConfig.fetchSharedConfig()
+
+    const initializedWallets = (await wallet.initializeWallets(
+      rawConfig
+    )) as InitializedWallets
+
+    const evmNetworks = rawConfig.domains.filter(
+      (domain) => domain.type === Network.EVM
+    ) as Array<EthereumConfig>
+
+    const substrateNetworks = rawConfig.domains.filter(
+      (domain) => domain.type === Network.SUBSTRATE
+    ) as Array<SubstrateConfig>
+
+    const rpcEndpoints = filesystem.read(
+      'rpcEndpoints.json',
+      'json'
+    ) as RpcEndpoints
+
+    if (evmNetworks) {
```

Comment:
If this is an array won't this always be true.
Though I think you don't even this check personally because if it is an empty array it would just skip anyway.